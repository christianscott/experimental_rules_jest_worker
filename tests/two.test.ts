test('two', () => {
  expect(2).toMatchSnapshot();
})
