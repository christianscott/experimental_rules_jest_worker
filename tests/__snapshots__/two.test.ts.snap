// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`two 1`] = `2`;
